test/e2e/kubectl/kubectl.go

```diff
@@ -364,6 +365,97 @@ var _ = SIGDescribe("Kubectl client", func() {
 		})
 	})
 
+	ginkgo.Describe("kubectl get output", func() {
+		objMeta := metav1.ObjectMeta{Name: "testing-kubectl-get-output"}
+		ns := "default"
+
+		ginkgo.BeforeEach(func() {
+			lists, err := c.Discovery().ServerPreferredResources()
+			framework.ExpectNoError(err)
+
+			if resourceEnabled(lists, "endpointslices") {
```

Comment:
Try using the content in the integration test https://github.com/kubernetes/kubernetes/blob/master/test/integration/etcd/data.go  to create namespace scoped resources to prime these values.  

this is a great idea for a test.

Reply:
Thanks for the idea! Unfortunately most of the data there does not have a Kind specified which makes it difficult to create in a real k8s cluster. I did actually go through and attempt to create all resources that had a `ExpectedGVK` set, but unfortunately a number of them failed validation. Although I like the idea of adding more resources here, I couldn't find a good way to make this work as a source.

With that said, I think this PR is still valuable because it will fail for each new resource added, and should encourage proper kubectl integration to start. Better test coverage for existing resources would certainly be great, but I think this is still valuable as is.

Reply:
@robscott I've created PR with a prototype impl: https://github.com/robscott/kubernetes/pull/2

Reply:
This is done differently, see https://github.com/kubernetes/kubernetes/blob/5f900f63327eb893dbf8bfa3ad2c1f54643e2148/test/e2e/kubectl/kubectl.go#L1677 which is easier and faster.